### DISCUSSION TITLE
Notify NewRelic when rollout complte

### DISCUSSION THREAD

**Discussion by igal-harel-sage** [argoproj/argo-rollouts#3599] [General]
Hi

Is it possible to use argo-rollouts notification to update newrelic when rollout has complete?
In case it is possible, is there a procedure how to implement it?

I checked [this](https://argo-rollouts.readthedocs.io/en/stable/generated/notification-services/newrelic/) but it's doesn't seem to work.

Thanks
Igal

**Comment by kostis-codefresh** [comment]
You also do a generic webhook to a different system that itself notifies Newrelic https://argo-rollouts.readthedocs.io/en/stable/generated/notification-services/webhook/

If the native integration with New Relic doesn't work, please open an issue with all the details.

**Reply by igal-harel-sage** [reply]
Thanks a log :)
